README.md

```diff
@@ -106,5 +106,9 @@ nix-shell \
   --argstr bkyAsVersion "latest" \
   --run "\
     YOUR_COINGECKO_API_KEY=${COINGECKO_API_DEV_KEY} \
+    YOUR_PANDASCORE_API_KEY=${PANDASCORE_API_DEV_KEY} \
```

Comment:
does YOUR_PANDASCORE_API_ENDPOINT also need to be added to this list of env vars? Alternatively, if this example is more of a burden than a help, we can delete the example or provide a different one.